cmd/payload-testing-ui/server.go

```diff
@@ -126,7 +128,18 @@ type server struct {
 }
 
 func newServer(client ctrlruntimeclient.Client, ctx context.Context, namespace string) (server, error) {
-	runsListTemplate, err := template.New("runsListTemplate").Parse(runsListTemplate)
+	runsListTemplate, err := template.New("runsListTemplate").Funcs(template.FuncMap{
+		"runLink": func(obj *metav1.ObjectMeta, pr *prpqv1.PullRequest) template.HTML {
+			objName := obj.GetName()
+			objNS := obj.GetNamespace()
+			ident := objName[len(objName)-6:]
+			author := template.HTMLEscapeString(pr.Author)
+			title := template.HTMLEscapeString(pr.Title)
+			ret := fmt.Sprintf(`<a href="`+runsURL+`%s/%s">%s-%s-%s</a>`, objNS, objName, author, title, ident)
```

Comment:
```suggestion
			ret := fmt.Sprintf(`<a href="%s%s/%s">%s-%s-%s</a>`, runsUrl, objNS, objName, author, title, ident)
```

Reply:
The separation with hyphens is not very clear (and is ambiguous).

We can try a table, as is done in: https://steps.ci.openshift.org.
That's implemented by: https://github.com/openshift/ci-tools/blob/master/pkg/webreg/webreg.go.